README.md

```diff
@@ -1,11 +1,41 @@
-AWS Concourse
-===
+# AWS Concourse
 
 An AWS based Concourse platform
 
 Built on top of [concourse-control-tower](https://github.com/dwp/concourse-control-tower)
 
+## Setting up on first use
+
+In order to install dependencies required by this repo, run `make bootstrap`
+
+## Deploying Concourse to Development Account
+
+```
+make boostrap-terraform-dev
```

Comment:
typo - boostrap -> bootstrap

Reply:
tab predict kept this hidden from me, thanks for spotting